classes/ActionScheduler_CanceledAction.php

```diff
@@ -6,10 +6,10 @@
  * Stored action which was canceled and therefore acts like a finished action but should always return a null schedule,
  * regardless of schedule passed to its constructor.
  */
-class ActionScheduler_CanceledAction extends ActionScheduler_FinishedAction {
+class ActionScheduler_CanceledAction extends ActionScheduler_NonexecutableAction {
```

Comment:
If possible, this should be `ActionScheduler_Canceled_Action` for consistency.

Reply:
Consistency with what? No other class in Action Scheduler uses an underscore to separate a word and action type. 🙂